### PR TITLE
Renamed spaces_around_equal to space_character.

### LINKED ISSUE #119: Rename check SpacesAroundEqual to SpaceCharacter
Need to rename the check `SpacesAroundEqual` to `SpaceCharacter`.

### PR DESCRIPTION
**Pull request by SaMuRa1ReM1X**
Fixes #119 

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [ ] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### 🚀 Added
+- Renamed check SpacesAroundEqual to SpaceCharacter [#134](https://github.com/mgrachev/dotenv-linter/pull/134) ([@SaMuRa1ReM1X](https://github.com/SaMuRa1ReM1X))
```
Comment (mgrachev) [inline CHANGELOG.md]: Please move this changes to the `Changed` block.
I guess you need to update your fork to get fresh changes for [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md).

**Comment by mgrachev** [comment]
@SaMuRa1ReM1X 👋Thank you for your contribution! 💪 

I have left a comment. Could you please fix it?

Also you have forgotten to update the [README.md](https://github.com/mgrachev/dotenv-linter#spaces-around-equal-sign) file.

File: CHANGELOG.md (hunk 1 above)
Comment (SaMuRa1ReM1X) [reply CHANGELOG.md]: Sorry about that, I'll move it to the Changed block after I update my fork.

**Comment by mgrachev** [comment]
@SaMuRa1ReM1X Thank you! 🖖

**Pull request merged**